core/domain/collection_domain.py

```diff
@@ -199,9 +207,19 @@ def skills(self):
         return set(self.prerequisite_skills) | set(self.acquired_skills)
 
     def update_prerequisite_skills(self, prerequisite_skills):
+        """Updates prerequisite skills of collection node.
+
+        Note: Please use the add_collection_node_prerequisite_skill
```

Comment:
Discussed during TL meeting: we want validation to be in the lowest common place, in this case in the domain object since there's no requirement for the services function to need to do any additional validation.